src/dotdom.js

```diff
@@ -90,15 +89,15 @@ module.exports = window;
                                                                       // placeholder for the local variables after
 
         _path=_npath+' '+index,                                       // a. The state path of this vnode
-        _path_state=wrapClassProxy[_path] || [{}, vnode.$],           // b. Get the state record for this path
+        _path_state=w[_path] || [{}, vnode.$],                        // b. Get the state record for this path
```

Comment:
@wavesoft check your build, and open the minified file, the names do not get compressed.